### ISSUE TITLE
One commit in a merged pull request is signed off with a wrong account

### ISSUE THREAD

**Issue by dilawar** [BhallaLab/moose-core#417]
PR: https://github.com/BhallaLab/moose-core/pull/410 
Commit: https://github.com/BhallaLab/moose-core/pull/410/commits/55cbb15cd64ac927d01aebade95f8c52eba51707 (Signed off as "Anushree Narajal")

This particular commit (https://github.com/BhallaLab/moose-core/pull/410/commits/55cbb15cd64ac927d01aebade95f8c52eba51707) was pushed using my account but signed as "Anushree Narjala" because of a shared system which has the ~/.gitconfig with her details. Hence her name does not appear in the list of contributors https://github.com/BhallaLab/moose-core/graphs/contributors

To fix this, admin needs to `amend` this commit (`git filter-branch`) and `--force-push-with-lease` to the master branch. Be careful not to use `--force` since it may invalidate all other forks.

**Comment by dilawar** [comment]
Such a case is discussed here https://www.git-tower.com/learn/git/faq/change-author-name-email

**Comment by hrani** [comment]
Let me know exact steps to change this, since its not most current commit if 
git commit --amend (change the name)
git push origin HEAD:master
! [rejected]            HEAD -> master (non-fast-forward)
and either I want to push force 
I am sure there is cleaner way to do this.

**Comment by dilawar** [comment]
I don't know any other way without rewriting history and without a `force push`.

**Comment by hrani** [comment]
Yes because it was squashed commit it was difficult to change just one commits author name I wanted to double sure before doing this. Will force the push and commit

**Comment by dilawar** [comment]
Ok. Probably a good idea to merge the PRs first and then rewrite the history. Else any of the PR *can* bring the commit back.

**Comment by hrani** [comment]
That exactly opposite of what I want to do, First change and start merging all the PRs. you can update the PRs later and re-run build

**Comment by hrani** [comment]
git commit --amend (editor the author name)

git push --force origin (wit my username and password) 

remote: This repository moved. Please use the new location:
remote:   https://github.com/BhallaLab/moose-core.git
remote: error: GH006: Protected branch update failed for refs/heads/master.
remote: error: Cannot force-push to this protected branch
To http://github.com/bhallalab/moose-core

Should I just enable "Allow force push" and push or there is any better way

**Comment by dilawar** [comment]
For time being, allow force on master branch and protect it again. I don't see any other way around it.

⁣Sent from a handheld device, please pardon my brevity and tpyos. 

Get BlueMail for Android ​

**Comment by hrani** [comment]
done
